czkawka_cli/src/commands.rs

```diff
@@ -0,0 +1,192 @@
+        number_of_files: usize,
+        #[structopt(flatten)]
+        file_to_save: FileToSave,
+        #[structopt(flatten)]
+        not_recursive: NotRecursive,
+    },
+    #[structopt(name = "empty-files", about = "Finds emtpy files", help_message = HELP_MESSAGE, after_help = "EXAMPLE:\n    czkawka empty-files -d /home/rafal /home/szczekacz -e /home/rafal/Pulpit -R -f results.txt")]
+    EmptyFiles {
+        #[structopt(flatten)]
+        directories: Directories,
+        #[structopt(flatten)]
+        excluded_directories: ExludedDirectories,
+        #[structopt(flatten)]
+        excluded_items: ExludedItems,
+        #[structopt(flatten)]
+        allowed_extensions: AllowedExtensions,
+        #[structopt(short = "D", long, help = "Delete found files")]
+        delete_files: bool,
+        #[structopt(flatten)]
+        file_to_save: FileToSave,
+        #[structopt(flatten)]
+        not_recursive: NotRecursive,
+    },
+    #[structopt(name = "temp", about = "Finds temporary files", help_message = HELP_MESSAGE, after_help = "EXAMPLE:\n    czkawka temp -d /home/rafal/ -E */.git */tmp* *Pulpit -f results.txt -D")]
+    Temporary {
+        #[structopt(flatten)]
+        directories: Directories,
+        #[structopt(flatten)]
+        excluded_directories: ExludedDirectories,
+        #[structopt(flatten)]
+        excluded_items: ExludedItems,
+        #[structopt(short = "D", long, help = "Delete found files")]
+        delete_files: bool,
+        #[structopt(flatten)]
+        file_to_save: FileToSave,
+        #[structopt(flatten)]
+        not_recursive: NotRecursive,
+    },
+}
+
+#[derive(Debug, StructOpt)]
+pub struct Directories {
+    #[structopt(short, long, parse(from_os_str), required = true, help = "Directorie(s) to search", long_help = "List of directorie(s) which will be searched(absolute path)")]
+    pub directories: Vec<PathBuf>,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct ExludedDirectories {
+    #[structopt(short, long, parse(from_os_str), help = "Exluded directorie(s)", long_help = "List of directorie(s) which will be excluded from search(absolute path)")]
+    pub excluded_directories: Vec<PathBuf>,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct ExludedItems {
+    #[structopt(short = "E", long, parse(from_os_str), help = "Exluded item(s)", long_help = "List of excluded item(s) which contains * wildcard(may be slow, so use -e where possible)")]
+    pub excluded_items: Vec<PathBuf>,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct AllowedExtensions {
+    #[structopt(
+        short = "x",
+        long,
+        help = "Allowed file extension(s)",
+        long_help = "List of checked files with provided extension(s). There are also helpful macros which allow to easy use a typcal extensions like:\nIMAGE(\"jpg,kra,gif,png,bmp,tiff,webp,hdr,svg\"),\nTEXT(\"txt,doc,docx,odt,rtf\"),\nVIDEO(\"mp4,flv,mkv,webm,vob,ogv,gifv,avi,mov,wmv,mpg,m4v,m4p,mpeg,3gp\") or\nMUSIC(\"mp3,flac,ogg,tta,wma,webm\")"
```

Comment:
Here probably should be 1 more empty line - 
![Zrzut ekranu z 2020-10-04 20-36-52](https://user-images.githubusercontent.com/41945903/95024122-69c50700-0681-11eb-92d1-ae68ce4cde66.png)